samples/nrf9160/serial_lte_modem/src/slm_util.c

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+#include <stdlib.h>
+#include <string.h>
+#include <errno.h>
+#include <stdio.h>
+#include "slm_util.h"
+
+#define PRINTABLE_ASCII(ch) (ch > 0x1f && ch < 0x7f)
+
+/**
+ * @brief Compare string ignoring case
+ */
+bool slm_util_cmp(const char *str1, const char *str2, u16_t str2_len)
```

Comment:
Do we need `len` parameter? IMO it's error-prone to require length of one string, and not the other. If they're truly strings, you can just use `strlen` in the function.

And I'd consider making it `slm_util_casecmp` to indicate it's case-insensitive - such nomenclature is used in linux for instance (https://linux.die.net/man/3/strcasecmp).

Reply:
Len removed, using strlen() inside the function to get target string length. Renamed to slm_util_casecmp()